src/pages/Donate/Donate.js

```diff
@@ -1,17 +1,15 @@
 import React from 'react';
-import { useDispatch } from 'react-redux';
-import { openModal } from './../../actions/index';
 
+import Button from './../../components/Button/Button';
 import CustomThankYou from './../../components/Custom/CustomThankYou/CustomThankYou';
 
 const Donate = () => {
-  const dispatch = useDispatch();
   const thankYou = <CustomThankYou />
 
   return (
     <div>
       <h4>Donate</h4>
-      <button onClick={() => dispatch(openModal(thankYou))}>Donate here!</button>
+      <Button type='filled' modal={thankYou}>Opens Modal</Button>
```

Comment:
imo it should keep dispatch logic here
```suggestion
      <Button type='filled' onClick={() => dispatch(openModal(thankYou))>Opens Modal</Button>
```